doc/Hammer-Use/index.rst

```diff
@@ -13,3 +13,4 @@ This documentation will walk through more advanced features of the Hammer infras
    Hooks
    Buildfile
    Hierarchical
+   Flowgraphs
```

Comment:
I'd prefer for Flowgraphs to come before Buildfile, since it should be the primary method to manage a flow. Should also mention in the documentation above (in the beginning) that if the user prefers to have a pre-generated flow, use the Makefile instead.